core/src/main/res/values-es/strings.xml

```diff
@@ -131,6 +131,8 @@
   <string name="do_not_ask_anymore">No preguntar de nuevo</string>
   <string name="your_languages">Idiomas seleccionados:</string>
   <string name="other_languages">Otros idiomas:</string>
+  <string name="open_new_tab">Abrir nueva pestaña</string>
```

Comment:
You should only modify strings.xml. The other languages will be translated via a different procedure.